packages/primitives/package.json

```diff
@@ -30,13 +30,13 @@
     "blob-polyfill": "7.0.20220408",
     "esbuild-plugin-alias": "latest",
     "event-target-shim": "6.0.2",
-    "formdata-node": "4.4.1",
+    "formdata-node": "5.0.0",
     "text-encoding": "0.7.0",
     "tsup": "6",
-    "undici": "5.11.0",
-    "urlpattern-polyfill": "6.0.1",
+    "undici": "5.13.0",
+    "urlpattern-polyfill": "6.0.2",
     "web-streams-polyfill": "4.0.0-beta.3",
-    "whatwg-url": "11.0.0"
+    "whatwg-url": "12.0.0"
```

Comment:
essentially dropping node12

Reply:
not related with the dependency upgrade, but CLI is not printing URL as expected:

```
ƒ => new URL('https://example.com')
URL {  }
ƒ => const url = new URL('https://example.com')
undefined
ƒ => url
URL {  }
ƒ =>
```